pkg/util/kubernetes/apiserver/apiserver_nocompile.go

```diff
@@ -54,3 +55,14 @@ func (c *APIClient) StartClusterMetadataMapping(_ chan struct{}) {
 	log.Errorf("StartClusterMetadataMapping not implemented %s", ErrNotCompiled.Error())
 	return
 }
+
+// StartAutoscalerController runs the Autoscaler controller to collect the HorizontalPodAutoscaler data.
+func StartAutoscalerController(LeaderElectorItf LeaderElectorItf, dogCl hpa.DatadogClient, _ chan struct{}) error {
+	log.Errorf("StartAutoscalerController not implemented %s", ErrNotCompiled.Error())
+	return
+}
```

Comment:
Do we need to have these function stubs in here yet since we have no situation for calling this function from a file that does not have the `kubeapiserver` build tag?

Reply:
correct